string class_label
float64 probability
string hypothesis
